### PR TITLE
feat: add doc comments from #5 (but without types)

### PR DESCRIPTION
**Pull request by mathe42**
Co-authored-by: @michaelprichardson

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -362,6 +425,11 @@ export default class Surreal extends Emitter {
 		});
 	}
 
+
```
Comment (tobiemh) [inline src/index.ts:428]: Blank line

File: src/index.ts [hunk 2]
```diff
@@ -310,6 +355,10 @@ export default class Surreal extends Emitter {
 		});
 	}
 
+		/**
```
Comment (tobiemh) [inline src/index.ts:358]: Indentation

File: src/index.ts [hunk 3]
```diff
@@ -259,6 +283,11 @@ export default class Surreal extends Emitter {
...
```
Comment (tobiemh) [inline src/index.ts:286]: Indentation

**Review by tobiemh — COMMENTED** [review]
Just nitpicks!

**Comment by mathe42** [comment]
Will wait on #21 to be merged. After that fixing the issues can be done with `deno fmt`.

**Comment by tobiemh** [comment]
Rebase and `deno fmt` and I'll merge @mathe42 !

**Comment by mathe42** [comment]
@tobiemh ready to merge!

**Pull request merged**